### ISSUE TITLE
C syntax highlighting not working as before

### ISSUE THREAD

**Issue by Elhodred** [c0r73x/neotags.nvim#34]
After some time I updated all my plugins in neovim and I found that some of the highlights are not working like cType and others 

<img width="562" alt="screen shot 2018-08-23 at 17 04 20" src="https://user-images.githubusercontent.com/5877961/44533947-eccfb100-a6f6-11e8-9137-5d1e2a47b165.png">

I'm using rakr/vim-one as my colour scheme I don't see any strange change there.
Anyone can point to me what to look for? I'm getting a bit crazy here

**Comment by c0r73x** [comment]
Yeah something is strange, i tested some things now and if I try to add for example "local" variables to the ```g:neotags#c#order``` it dosen't pick it up. The tag file from neotags still don't output local variables. And if i try to force if with running a tag file in the source folder i get a strange error "Neotags: No lang/kind".

I will investigate some more, or maybe @roflcopter4 have an answer for this strangeness?

**Comment by roflcopter4** [comment]
I have to admit that I haven't even looked at the code in a few months, I've been writing what is basically a version of this plugin entirely in C (why? because I'm a masochist). Obvious plug here, but if neotags is semi-broken at the moment it might be worth looking at. Right now it is faster and does more than neotags (like actually parsing headers for relevant tags, reading .json files, running things in parallel, and directly talking to neovim without an api in between). [Anyway, if you're interested (and don't mind the occasional segfault) it's here.](https://github.com/roflcopter4/tag-highlight.nvim) *ENDPLUG*

Otherwise I suppose I really ought to take a look at this problem. I can't really imagine why default highlighting groups would be getting clobbered.

As to the customizing problems, I'll take a look too. The way the code worked required ctags to be run with particular arguments and was picky about the order. Adding to the `g:neotags#c#order` variable won't make the plugin actually run ctags with the arguments to output those tags, and if you run ctags manually you have to be certain that you got the arguments correct (which is why the code couldn't find the language or kind field in your case).

**Comment by roflcopter4** [comment]
I think I have at least partially solved this problem (by throwing away 75% of the code and rewriting it...). Since the rewrite doesn't implement everything yet (C code is 100% broken, some features are ignored, gzip only, etc) I am keeping it in the dev branch, but everything seems to work as it should. If anyone wants to test it that would be helpful.

**Comment by darosior** [comment]
FWIW, and if you land here googling the "`Neotags: No lang/kind`" : the dev branch solved it for me. If you are using Plug :
```vim
Plug 'c0r73x/neotags.nvim', { 'branch': 'dev', 'do': ':UpdateRemotePlugins' }
```
Thank you @roflcopter4 !
